src/components/charts/SummaryBarChart.jsx

```diff
@@ -0,0 +1,100 @@
+import {
+  ResponsiveContainer, BarChart, Bar,
+  Tooltip, XAxis, YAxis, LabelList, Cell,
+} from 'recharts';
+import PropTypes from 'prop-types';
+import React from 'react';
+import ChartsHelper from './ChartsHelper';
+import './SummaryBarChart.less';
+
+const yAxisStyle = {
+  fontSize: '12px',
+  fontWeight: 'regular',
+  lineHeight: '1em',
+  letterSpacing: '.03rem',
+  color: '#606060',
+};
+
+const labelValueStyle = {
+  fontSize: '10px',
+  fontWeight: 600,
+  lineHeight: '1em',
+  letterSpacing: '.02rem',
+  color: '#3283c8',
+};
+
+class SummaryBarChart extends React.Component {
+  render() {
+    const monoFillColor = this.props.monoColor ? this.props.color : undefined;
+    const barChartHeight = (this.props.data.length * this.props.barSize)
+      + ((this.props.data.length + 1) * this.props.barGap) + 2;
+    const barChartData = ChartsHelper.calculateBarChartData(
+      this.props.data,
+      this.props.showPercentage,
+      this.props.percentageFixedPoint,
+    );
+    const dataKey = ChartsHelper.getDataKey(this.props.showPercentage);
+    return (
+      <div className="summary-bar-chart">
+        <div className="summary-bar-chart__title h4-typo">
+          {this.props.title}
+        </div>
+        <ResponsiveContainer width="100%" height={barChartHeight}>
+          <BarChart
+            layout={this.props.vertical ? 'vertical' : 'horizontal'}
+            data={barChartData}
+            barCategoryGap={this.props.barGap}
+            barSize={this.props.barSize}
+            margin={{ top: 4, right: 35, left: 15 }}
+          >
+            <Tooltip formatter={ChartsHelper.percentageFormatter(this.props.showPercentage)} />
+            <XAxis axisLine={false} tickLine={false} type="number" hide />
+            <YAxis axisLine={false} tickLine={false} dataKey="name" type="category" style={yAxisStyle} interval={0} />
+            <Bar dataKey={dataKey}>
+              {
+                barChartData.map((entry, index) => (
+                  <Cell
+                    key={dataKey}
+                    fill={monoFillColor
```

Comment:
instead of doing the calculation for `const monoFillColor` on line 28, can this just be `fill={this.props.color || ChartsHelper.getCategoryColor(...)}` ? I don't love setting something to be undefined like its doing on line 28